README.md

```diff
@@ -85,6 +85,12 @@ Some rules require options. There are no default values, so each rule that requi
 }
 ```
 
+#### Shareable configs
+
+If you prefer to enforce a third-party styleguide (rather than craft your own config), you can use:
+
+* [SuitCSS shareable config](https://github.com/stylelint/stylelint-config-suitcss)
+
```

Comment:
How about adding something like "You can also extend a third-party config file, starting with what's there and making your own modifications and additions."